### ISSUE TITLE
Citations in footnotes

### ISSUE THREAD

**Issue by vincenzo** [dsifford/academic-bloggers-toolkit#590]
Would it be much work on your side to allow citations in footnotes? 
I am guessing it would entail making the footnotes editable in blocks, so that citations can be entered just as they are in other blocks.

**Comment by dsifford** [comment]
Unfortunately supporting citations in footnotes would require more time and effort than I have available in the near future. Doing so would require a fundamental rewrite of how the footnotes work and backwards compatibility would be tricky.

**Comment by vincenzo** [comment]
@dsifford I was thinking, in that case, that it would be useful if the footnotes system provided by your plugin was optional, and could be disabled with a setting. 

The idea behind this is that I am personally willing to use a mixture of plugins, yours for academic references, and another for footnotes. And some other people might like that too. However, when I do, I get tow "Add footnote" items in the block's menu. Though I could eventually learn which one is the one I want, it'd be great to see only one :D 

What do you think?

**Comment by dsifford** [comment]
Fair enough. I'll reopen this for that feature request, but can't give an estimation on when I'm going to be able to get to it.

**Comment by vincenzo** [comment]
Thanks. If I can navigate the code, I’ll try and do it myself, and issue a PR.

**Comment by vincenzo** [comment]
@dsifford 

Hi. I have interest in this plugin being kept alive and developed. 
What would you need to commit some of your monthly time to the project?

Happy to discuss in private.

**Comment by dsifford** [comment]
Feel free to email if you'd like: dereksifford@gmail.com

Ideally, the best solution for more updates/maintenance for this project would be to have more maintainers.

I've switched jobs last year and I no longer use this plugin regularly like I used to.
